src/frontend/src/components/RecordSearch/SearchPanel/index.tsx

```diff
@@ -117,7 +117,7 @@ export default class SearchPanel extends React.Component<Props, State> {
     });
     return (
       <div>
-        <h1 className="f4 fw6 tc mv4">Record Search</h1>
+        <h1 className="visually-hidden">Record Search</h1>
```

Comment:
removing this looks good. is there a reason not to just delete the element?

Reply:
Just best practice for every view to have an h1, potentially for screen readers

Reply:
gotcha